### PR TITLE
Why3 1.4.1

### PR DESCRIPTION
**Pull request by silene**

### CODE REVIEW

**Comment by silene** [comment]
Of the 39 failed jobs, 37 are caused by a failure to install Gtk2 and 1 is caused by a network error. The remaining failure is with the compiler `4.12.0+domains+effects`, as it defines extra OCaml keywords which conflict with Why3 source code.

File: packages/why3/why3.1.4.1/opam [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  "ppx_deriving" {build}
+  "ppx_sexp_conv" {build}
+  "mlmpfr"
+]
+
+conflicts: [
+  "why3-base"
+  "ocamlgraph" {< "1.8.2"}
+  "mlmpfr" {< "4.0.0"}
+]
```
Comment (mseri) [inline packages/why3/why3.1.4.1/opam:66]:
```suggestion
  "base-effects"
]
```

**Comment by silene** [comment]
Thanks for suggesting `base-effects`. I was looking for something akin to `ocaml-option-effects` and did not find anything relevant.

(The network error "Could not resolve host: github.com" is still there, so it seems like it is not transient but specific to the 4.14 job.)

**Comment by mseri** [comment]
Yes, don't worry about that. The rest looks fine

**Pull request merged**